flags/sources.py

```diff
@@ -33,22 +33,23 @@ class Flag(object):
 
     def __init__(self, name, conditions=[]):
         self.name = name
-        self.conditions = conditions
-        self.optional_conditions = [
-            c for c in self.conditions if not c.required
-        ]
-        self.required_conditions = [
-            c for c in self.conditions if c.required
-        ]
+        self.__conditions = conditions
 
     def __eq__(self, other):
         """ There can be only one feature flag of a given name """
         return other.name == self.name
 
     def check_state(self, **kwargs):
         """ Determine this flag's state based on any of its conditions """
-        if (len(self.optional_conditions) == 0
-                and len(self.required_conditions) == 0):
+        optional_conditions = [
+            c for c in self.conditions if not c.required
```

Comment:
Shouldn't these lines refer to `self.__conditions`? Is there still a `self.conditions`?

Reply:
Oh, oops. that was a separate change I meant to revert. `self.conditions` should be `self.conditions`!

Reply:
Fixed in 5c275d8.